generators/client/templates/vue/src/main/webapp/app/admin/user-management/user-management.component.ts.ejs

```diff
@@ -1,11 +1,11 @@
-import { mixins } from 'vue-class-component';
-import { Component, Inject } from 'vue-property-decorator';
+import { Component, Inject, Vue } from 'vue-property-decorator';
 import UserManagementService from './user-management.service';
-import Principal from '../../account/principal';
+import Principal from '@/account/principal';
 
 @Component
-export default class <%=jhiPrefixCapitalized%>UserManagementComponent extends mixins(Principal) {
+export default class JhiUserManagementComponent extends Vue {
```

Comment:
Why did you remove the templated prefix?

Reply:
I didn't do it on purpose! Thanks :)